server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreStatsIT.java

```diff
@@ -286,15 +286,18 @@ public void testDownloadStatsCorrectnessSinglePrimarySingleReplica() throws Exce
             zeroStatePrimaryStats.uploadBytesStarted,
             zeroStatePrimaryStats.uploadBytesFailed
         );
-        assertTrue(
-            zeroStatePrimaryStats.totalUploadsStarted == zeroStatePrimaryStats.totalUploadsSucceeded
-                && zeroStatePrimaryStats.totalUploadsSucceeded == 1
-        );
-        assertTrue(
-            zeroStatePrimaryStats.uploadBytesStarted == zeroStatePrimaryStats.uploadBytesSucceeded
-                && zeroStatePrimaryStats.uploadBytesSucceeded > 0
-        );
-        assertTrue(zeroStatePrimaryStats.totalUploadsFailed == 0 && zeroStatePrimaryStats.uploadBytesFailed == 0);
+
+        assertBusy(() -> {
+            assertTrue(
+                zeroStatePrimaryStats.totalUploadsStarted == zeroStatePrimaryStats.totalUploadsSucceeded
+                    && zeroStatePrimaryStats.totalUploadsSucceeded == 1
+            );
+            assertTrue(
+                zeroStatePrimaryStats.uploadBytesStarted == zeroStatePrimaryStats.uploadBytesSucceeded
+                    && zeroStatePrimaryStats.uploadBytesSucceeded > 0
+            );
+            assertTrue(zeroStatePrimaryStats.totalUploadsFailed == 0 && zeroStatePrimaryStats.uploadBytesFailed == 0);
+        }, 60, TimeUnit.SECONDS);
```

Comment:
Let's reduce the `assertBusy` wait time. Ideally the default value of this method should suffice, let's remove the extra time value we are adding to this `assertBusy` invocation

Reply:
Got it. Keeping the default value of asserBusy() which is 10 seconds.